.github/workflows/test-e2e.yml

```diff
@@ -218,6 +236,9 @@ jobs:
           # one job may wait for deploys in other jobs (only one deploy may be in progress for
           # a given alias at a time), resulting in cascading timeouts.
           DEPLOY_ALIAS: vercel-next-e2e-${{ matrix.version_spec.selector }}-${{ matrix.group }}
+          NEXT_RESOLVED_VERSION: ${{ matrix.version_spec.version }}
+          IS_WEBPACK_TEST: ${{ steps.decide-default-bundler.outputs.default_bundler == 'webpack' && '1' || '' }}
+          IS_TURBOPACK_TEST: ${{ steps.decide-default-bundler.outputs.default_bundler == 'turbopack' && '1' || '' }}
```

Comment:
those env vars are used by next test helpers and will impact `isTurbopack` etc checks - for example like one used in this test https://github.com/vercel/next.js/blob/a7d3aecafedc920221a885b8ab00f35f6de5e91c/test/e2e/esm-externals/esm-externals.test.ts